lib/brains/views/method.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from lib.brains.client import get_client
+from lib.brains.errors import BraintreeResultError
+from lib.brains.forms import PaymentMethodForm
+from lib.brains.serializers import (
+    PaymentMethodSerializer as BraintreeSerializer)
+from lib.buyers.models import PaymentMethod
+from lib.buyers.serializers import PaymentMethodSerializer
+from lib.transactions.constants import PROVIDER_BRAINTREE
+from solitude.constants import PAYMENT_METHOD_CARD
+from solitude.logger import getLogger
+
+
+log = getLogger('s.brains')
+
```

Comment:
Because this way lies madness and some of the craziness that occurs in Bango and Boku where endpoints become the combination of two competing needs and functionalities. For example a serializer in DRF is used to read the data and output the data, its also pretty closely tied to a model and a serializer. If you get an end point that does multiple things you get some of the methods like Bango which currently have a form to process Solitude data and then a serializer to get Bango data and it gets confusing as hell.

After learning from that this feels really clean to me. You've got a way of querying solitude easily. But you can't change much from that, because really its controlled by the provider.

If we did get payment methods for Bango, we'd just add in a similar method for Bango and we'd be done. This approach lets us write a really tailored view for what Braintree needs and make it nice and clean for Payments for FxA. Something that you can see we suffered from in webpay.

Ok long answer, sorry.